src/Uno.UI/Media/StreamGeometry.cs

```diff
@@ -51,10 +52,15 @@ internal void Close(Path bezierPath_)
 #if __SKIA__
 		internal override Path GetGeometrySource2D()
 		{
+			bezierPath.Geometry.FillType = FillRule.ToSkiaFillType();
 			return bezierPath;
 		}
 
-		internal override SKPath GetSKPath() => bezierPath.Geometry;
+		internal override SKPath GetSKPath()
```

Comment:
Can you cover this fix with a runtime test? E.g. draw a large colorful shape, screenshot and then assert if there is a color in the expected location?

Reply:
@MartinZikmund Thank you for your review, and I am currently learning how to write unit tests for UNO.

Reply:
@lindexi, in case you haven't seen this guide for adding tests to Uno, it's at https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html. Anyway, if you need help, please don't hesitate to let us know.

Reply:
Thank you @MartinZikmund and @ajpinedam 

Could you review my code. And I cover this fix by the unit test and the `SamplesApp` project.

Before fix:

![](https://github.com/unoplatform/uno/assets/16054566/c581806a-edf3-4552-8628-e5520b4f2524)

You can find the EventOdd FillRule will do nothing.

After this fix:

![](https://github.com/unoplatform/uno/assets/16054566/5fb9df7f-b3e3-46eb-81bc-b5c99706e71c)

You can find the EventOdd work well as WinUI platform.